spec/models/calculation_result_spec.rb

```diff
@@ -62,4 +62,30 @@
       expect(described_class.new(data).client_additional_property_rows.keys).to include(:deductions)
     end
   end
+
+  describe "#additional_vehicle_rows" do
+    it "can cope with multiple vehicles" do
+      data = FactoryBot.build(
+        :api_result,
+        vehicles: FactoryBot.build_list(:vehicle_api_result, 2),
+      )
+
+      expect(described_class.new(data).additional_vehicle_rows).not_to be_blank
+      expect(described_class.new(data).additional_vehicle_assessed_value).not_to be_blank
```

Comment:
think it would be nice to assert what's in here rather than just not blank

Reply:
would it be better to assert the keys contained in the hash or the actual values associated with the keys? Asserting the keys makes more sense to me, but I could easily be persuaded I am wrong.